### ISSUE TITLE
Possible Tag names to be searchable ?

### ISSUE THREAD

**Issue by behindthebid** [apsislabs/administrate-field-acts_as_taggable#1]
When adding a tags to the `AttributeType` it would be cool to be able to specify them as searchable in the Administrate dashboard.

Something like
```ruby
class SituationDashboard < Administrate::BaseDashboard
  ATTRIBUTE_TYPES = {
    tags: Field::ActsAsTaggable.with_options(searchable: true, searchable_field: 'name')
```

Currently, this crashes when doing a search with
`PG::UndefinedColumn: ERROR:  column situations.tags does not exist`

**Comment by wkirby** [comment]
@behindthebid I agree, that would be cool — I'll have to dig into the Administrate API to see how much we get to control the searchable bit. As far as I'm aware, `searchable: true` takes the symbol from the `ATTRIBUTE_TYPES` hash and interpolates it into the DB query, and we don't get much say in how that works.

I'll try to find some time this week to look into it.

**Comment by behindthebid** [comment]
awesome, looking forward to hearing what you can dig up 👍
